README.md

```diff
@@ -0,0 +1,142 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: setup and mobile first Exercise</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+ 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] Kindly, Include **all headers** from the [README template](https://github.com/microverseinc/readme-template#tech-stack) Microverse provided because having a complete README will help other potential recruiters to understand & use your project.

- You're only able to remove the _Live Demo_ section unless deployment is not a project requirement & the _FAQ_ section.